src/core/templates/buttons.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="btn__response-container">
 
-  <div class="btn__marking js-btn-marking u-display-none" aria-label="{{#if _isCorrect}}{{@root/_globals._accessibility._ariaLabels.answeredCorrectly}}{{else}}{{@root/_globals._accessibility._ariaLabels.answeredIncorrectly}}{{/if}}">
+  <div class="btn__marking js-btn-marking u-display-none" aria-label="{{#all _isInteractionComplete _canShowMarking}}{{#if _isCorrect}}{{@root/_globals._accessibility._ariaLabels.answeredCorrectly}}{{else}}{{@root/_globals._accessibility._ariaLabels.answeredIncorrectly}}{{/if}}{{/all}}">
```

Comment:
could the getter be used here instead?
```suggestion
  <div class="btn__marking js-btn-marking u-display-none" aria-label="{{#if canShowMarking}}{{#if _isCorrect}}{{@root/_globals._accessibility._ariaLabels.answeredCorrectly}}{{else}}{{@root/_globals._accessibility._ariaLabels.answeredIncorrectly}}{{/if}}{{/all}}">
```

Reply:
No. The data in the handlebars content is not the model it's just a json object from model.attributes / model.toJSON() so there is no reference to the canShowMarking property.